com.unity.shadergraph/Editor/Data/Nodes/Utility/SubGraphNode.cs

```diff
@@ -157,7 +157,7 @@ public SubGraphAsset asset
 
         public override bool hasPreview
         {
-            get { return asset != null; }
+            get { return true; }
```

Comment:
preview displays black if there is no subgraph asset.   Forcing hasPreview to false leaves the view in a bad state (and it is weird in the UI if it auto-collapses the preview when the subgraph is missing).